tests/Old/CommonExtractorTest.php

```diff
@@ -798,27 +817,38 @@ public function testIncrementalFetchingDisabled(): void
         $this->assertEmpty($result['state']);
     }
 
-    public function testIncrementalFetchingInvalidColumns(): void
+    public function testIncrementalFetchingOnInvalidColumnName(): void
     {
         $this->createAutoIncrementAndTimestampTable();
         $config = $this->getIncrementalFetchingConfig();
         $config['parameters']['incrementalFetchingColumn'] = 'fakeCol'; // column does not exist
 
-        try {
-            $result = ($this->getApp($config))->run();
-            $this->fail('specified autoIncrement column does not exist, should fail.');
-        } catch (UserException $e) {
-            $this->assertStringStartsWith("Column [fakeCol]", $e->getMessage());
-        }
+        $app = $this->getApp($config);
+
+        $this->expectException(UserException::class);
+        $this->expectExceptionMessage(
+            'Column [fakeCol] specified for incremental fetching was not found in the table'
+        );
+        $app->run();
+    }
+
+    public function testIncrementalFetchingOnInvalidColumnAttributes(): void
+    {
+        $this->createAutoIncrementAndTimestampTable();
+        $config = $this->getIncrementalFetchingConfig();
+        $config['parameters']['incrementalFetchingColumn'] = 'fakeCol'; // column does not exist
```

Comment:
nechápu - na ř 842 se to přepíše...

Reply:
Tady jsem jen rozdelil puvodni metodu a tohle mi uniklo 👍